core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -77,29 +77,46 @@
+                    <div class="nav-content d-flex flex-column">
+                      <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})" (click)="navigateToClassroomPage('/learn/math')" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link nav-item-top-link" href="#" [innerHTML]="'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate"></a>
+                      <div class="des" [innerHTML]="'I18N_TOPNAV_CLASSROOM_DESCRIPTION_1' | translate"></div>
+                    </div>
+                    <picture>
+                      <source type="image/png"
+                              [srcset]="getStaticImageUrl('/topnav/learn.png')">
+                      <img [src]="getStaticImageUrl('/topnav/learn.png')"
+                           class="learn"
+                           alt="Learn" >
+                    </picture>
+                  </div>
+                  <div class="nav-item-bottom d-flex flex-column" *ngIf="!userIsLoggedIn">
+                    <div class="nav-item-links"  *ngFor="let topicSummary of classroomData.getTopicSummaries().slice(0, 3)">
+                      <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})" (click)="navigateToClassroomPage('/learn/math/' + topicSummary.getUrlFragment())">
+                      {{ topicSummary.getName() }}
+                      </a>
+                    </div>
+                    <a class="nav-item-bottom-link" href="/learn/math"><span [innerHTML]="'I18N_TOPNAV_CLASSROOM_LINK_1' | translate"></span> <span class="material-icons">arrow_right_alt</span></a>
+                  </div>
+                  <a *ngIf="userIsLoggedIn" class="nav-item-bottom-link-logged" href="/learn/math"><span [innerHTML]="'I18N_TOPNAV_CLASSROOM_LINK_2' | translate"></span> <span class="material-icons">arrow_right_alt</span></a>
+                </div>
+                <div class="nav-content nav-item-right d-flex flex-column">
+                  <div class="nav-item-right-head" [innerHTML]="'I18N_TOPNAV_CLASSROOM_HEADING' | translate"></div>
+                  <div class="des" [innerHTML]="'I18N_TOPNAV_CLASSROOM_DESCRIPTION_2' | translate"></div>
+                  <a class="nav-item-bottom-link" href="/community-library"><span [innerHTML]="'I18N_TOPNAV_LIBRARY' | translate"></span> <span class="material-icons">arrow_right_alt</span></a>
+                </div>
+              </div>
             </li>
           </ul>
         </li>
-        <li [hidden]="!navElementsVisibilityStatus.I18N_TOPNAV_LIBRARY"
-            class="nav-item oppia-clickable-navbar-element">
-          <a class="nav-link oppia-navbar-tab oppia-navbar-tab-content"
-             href="/community-library"
-             [routerLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.LIBRARY_INDEX.ROUTE"
-             [innerHTML]="'I18N_TOPNAV_LIBRARY' | translate">
-          </a>
-        </li>
```

Comment:
When classroom is not enabled, we should have an option to go to the library. But currently we will only have it when classroom is enabled. We should have 'learn' nav item always present, irrespective of classroom being enabled or not.
'Math foundations' column should be visible only when classroom is enabled, however 'community library' section should be always visible. (even when user is not signed in). Does that make sense?

Reply:
Done. Added screenshot in PR description

Reply:
thanks!